packages/client/package.json

```diff
@@ -48,6 +48,7 @@
     "@xrengine/matchmaking": "^0.4.13",
     "@xrengine/projects": "^0.4.13",
     "classnames": "2.3.1",
+    "ethereal": "^3.0.3",
```

Comment:
please ensure this is up to date with the version in the engine, currently 3.0.11